src/components/Auth/SignUpForm.js

```diff
@@ -6,27 +6,26 @@ import axios from 'axios';
 import { useAuth } from './AuthContext';
 import { validationResolver, defaultValues } from './SignUpForm.schema';
 import { Form, Field } from '../form';
+import { config } from '../../helpers/constants';
 
 const SignUpForm = ({ toggleActiveForm }) => {
   const [errorMessage, setErrorMessage] = useState(null);
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const referer = '/profile';
   const auth = useAuth();
 
-  const onSubmit = ({ username, password, firstName, lastName, email }) => {
+  const onSubmit = ({ username, password1, password2, email }) => {
     const data = {
       username,
-      password,
       email,
-      first_name: firstName,
-      last_name: lastName,
+      password1,
+      password2,
     };
     axios
-      .post('/auth/users/', data)
+      .post(`${config.API_URL}/api/v1/auth/registration/`, data)
       .then(res => {
         auth.setAuthTokens(res.data);
         setIsLoggedIn(true);
```

Comment:
Instead of setting `setIsLoggedIn(true);` and redirecting the user to `/profile`, perhaps we should remove this and redirect the user to the /verifyemail page instead. (See: line 14 above)